patterns/footer.php

```diff
@@ -113,7 +113,7 @@
       <div class="wp-block-group">
         <!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained","contentSize":"1000px","justifyContent":"center"}} -->
         <div class="wp-block-group">
-          <!-- wp:site-title {"textAlign":"left","isLink":false,"style":{"typography":{"letterSpacing":"0.02em","fontSize":"100px","lineHeight":"1"},"spacing":{"margin":{"top":"0px"},"padding":{"top":"0px"}}},"className":"footer-title","fontFamily":"sorts-mill-goudy"} /-->
+          <!-- wp:site-title {"level":0,"textAlign":"left","isLink":false,"style":{"typography":{"letterSpacing":"0.02em","fontSize":"100px","lineHeight":"1"},"spacing":{"margin":{"top":"0px"},"padding":{"top":"0px","left":"0"}}},"className":"footer-title","fontFamily":"sorts-mill-goudy"} /-->
```

Comment:
Looks good, not sure if it's a part of the requirement, is there a particular reason we are adding the left padding here?

Reply:
@Imran92  I removed all padding settings here [e382471](https://github.com/Automattic/course/pull/154/commits/e382471d34bd2375e8a739b6f7ca72176ea01d86)